packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -5,7 +5,9 @@
   >
     <a
```

Comment:
The NavLink element should be <a> only when it acts as a link (e.g. has a valid `href` value). If it's acting as a trigger to open a dropdown navigation, it should be a <button>. This should probably remove the need to set an explicit tabindex (<a> without href is not focusable, but <button> is focusable, at least on Chrome).